File: FitProj/app/src/main/java/com/teuskim/fitproj/common/DistanceSet.java
package com.teuskim.fitproj.common;

/**
 * 거리 정보
 */
public class DistanceSet {

    private float walk;
    private float run;
    private float cycle;

    public float getWalk(boolean isConvertUnitOn) {
        if (isConvertUnitOn) {
            return FitUtil.convertToMi(walk);
        }
        return walk;
    }

    public void setWalk(float walk) {
        this.walk = walk;
    }

    public float getRun(boolean isConvertUnitOn) {
        if (isConvertUnitOn) {
            return FitUtil.convertToMi(run);
        }
        return run;
    }

    public void setRun(float run) {
        this.run = run;
    }

    public float getCycle(boolean isConvertUnitOn) {
        if (isConvertUnitOn) {
            return FitUtil.convertToMi(cycle);
        }
        return cycle;
    }

    public void setCycle(float cycle) {
        this.cycle = cycle;
    }
}

File: FitProj_kotlin/app/src/main/java/com/teuskim/fitproj/common/DistanceSet.kt
package com.teuskim.fitproj.common

/**
 * 거리 정보
 */
class DistanceSet {

    private var walk: Float = 0.toFloat()
    private var run: Float = 0.toFloat()
    private var cycle: Float = 0.toFloat()

    fun getWalk(isConvertUnitOn: Boolean): Float {
        return if (isConvertUnitOn) {
            FitUtil.convertToMi(walk)
        } else walk
    }

    fun setWalk(walk: Float) {
        this.walk = walk
    }

    fun getRun(isConvertUnitOn: Boolean): Float {
        return if (isConvertUnitOn) {
            FitUtil.convertToMi(run)
        } else run
    }

    fun setRun(run: Float) {
        this.run = run
    }

    fun getCycle(isConvertUnitOn: Boolean): Float {
        return if (isConvertUnitOn) {
            FitUtil.convertToMi(cycle)
        } else cycle
    }

    fun setCycle(cycle: Float) {
        this.cycle = cycle
    }
}
